PopupBridge/src/main/java/com/braintreepayments/popupbridge/PopupBridge.java

```diff
@@ -125,6 +125,14 @@ public void onBrowserSwitchResult(int requestCode, BrowserSwitchResult result, U
             } catch (JSONException ignored) {}
 
             payload = json.toString();
+        } else if (result == BrowserSwitchResult.CANCELED) {
```

Comment:
Is there a difference in behavior here between the iOS PR and this? It seems this will call `onComplete` twice or `onClose()` and `onComplete` if it exists.

Reply:
How does it call it twice? It has an early return so it doesn't call it outside of the if block.

Reply:
Ahh I missed the `return` statement.